bundles/org.openhab.binding.clementineremote/README.md

```diff
@@ -0,0 +1,334 @@
+# ClementineRemote Binding
+
+<img style="float: right; margin: 0 0 5px 5px" src="doc/screenshot.png" alt="Screenshot" />
+This binding shall bring the benefits of over-the-network control of the [Clementine-Player] to OpenHab!
+
+It allows control of playback, position and volume and allows display of various title information:
+
+* Artist
+* Album
+* Title
+* current position within track
+* Album cover
+
+
+[→ useful Links][links]
+
+## Discovery
+
+This thing does (currently) not provide auto-discovery.
+You will have to manually configure clementine to allow remote connections.
+And you will have to manually configure the Thing instances to talk to your player:
+
+## Thing Configuration
+
+Configuring the connection to Clementine is quite easy:
+
+Normally you only need to set the hostname your Clementine player is running at.
+If you set a password in the Clementine configuration, you will have to provide it to the configuration, too.
+
+### `sample` Thing Configuration
+
+| Name     | Type    | Description                                                       | Default | Required | Advanced |
+|----------|---------|-------------------------------------------------------------------|---------|----------|----------|
+| hostname | text    | Hostname or IP address of the device                              | N/A     | yes      | no       |
+| password | text    | Password to access the device                                     | N/A     | no       | no       |
+| port     | integer | Port at which Clementine is listening for remote control commands | 5500    | yes      | no       |
+
+## Channels
+
+The binding provides the following channels:
+
+| Channel          | Type          | Read/Write  | Description                                  |
+|------------------|---------------|-------------|----------------------------------------------|
+| album            | String        | read-only   | Album the currently playing song belongs to  |
+| artist           | String        | read-only   | Artist of the currently playing title        |
+| cover            | Image         | read-only   | Cover of the current album                   |
+| playback-control | String        | read/write  | Common control of playback and position      |
```

Comment:
I think this should be Player

Reply:
Not sure. I'm not using the media-contol channel type, as described above. Thus my control accepts String commands. Anyways, I'd be happy to incooperate improvements.